viewing_party/party.py

```diff
@@ -0,0 +1,171 @@
+#WAVE 1
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        movie = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    updated_data = user_data["watched"]
+    updated_data.append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
```

Comment:
great implementation! you could also make it a one-liner user_data["watchlist"].append(movie)